src/lab_sim/objectives/clear_previous_obstacles.xml

```diff
@@ -0,0 +1,9 @@
+<root BTCPP_format="4" main_tree_to_execute="Clear Previous Obstacles">
```

Comment:
Should this Objective be added to `picknik_ur_base_config`? Seems like it would be useful for a number of derivative configs.

Reply:
I think this belongs in the `Perception - Planning Scene` category.

Reply:
What do you think of the name? Is it normal to have to do both?